src/legacy/Routes.tsx

```diff
@@ -11,6 +11,7 @@ import Dashboard from './components/Dashboard'
 import Send from './components/Send'
```

Comment:
We will have to move changes from here to the new Routes file, defined here: https://github.com/pastelnetwork/pastel-electron-wallet/blob/d48c4a7947e3af23f59990c5f4e706fbfcdde639/src/common/routes/Routes.tsx.